src/Features/CSharp/Portable/Structure/Providers/NamespaceDeclarationStructureProvider.cs

```diff
@@ -26,9 +26,13 @@ internal class NamespaceDeclarationStructureProvider : AbstractSyntaxNodeStructu
             if (!namespaceDeclaration.OpenBraceToken.IsMissing &&
                 !namespaceDeclaration.CloseBraceToken.IsMissing)
             {
+                SyntaxNodeOrToken current = namespaceDeclaration;
+                var nextSibling = current.GetNextSibling();
+
                 spans.AddIfNotNull(CSharpStructureHelpers.CreateBlockSpan(
                     namespaceDeclaration,
                     namespaceDeclaration.Name.GetLastToken(includeZeroWidth: true),
+                    compressEmptyLines: !nextSibling.IsNode || nextSibling.IsKind(SyntaxKind.NamespaceDeclaration),
```

Comment:
this feels unexpected as well.  types/namespaces are ok to spread out imo

Reply:
I reverted this part.